src/components/account_overview/AccountOverview.stories.mdx

```diff
@@ -216,7 +228,7 @@ export const Default = ({
 | details.max_approved_credit_limit_cents    | number | undefined |
 | details.interest_rate_percent              | number | undefined |
 | details.available_credit_cents             | number | undefined |
-| details.pending_charges                    | number | undefined |
+| details.pending_charges_cents              | number | undefined |
```

Comment:
I just noticed this is a breaking change. Can we create a CHANGELOG.md file and mention this and the object-configuration changes for AccountOverview?

Reply:
Copy. Should I revert it to pending_charges, then?

Reply:
Don't revert. Breaking changes are okay this early if as long as we're transparent about them in the CHANGELOG.md